js/src/common/utils/RequestError.ts

```diff
@@ -1,21 +1,21 @@
 export default class RequestError {
-  status: string;
+  status: string | number;
```

Comment:
Shouldn't these always be `number`? I know we currently use strings, but I'm not sure that's actually correct.